src/ui.jl

```diff
@@ -118,7 +122,7 @@ function TerminalMenus.keypress(m::CthulhuMenu, key::UInt32)
     elseif key == UInt32('b')
         m.toggle = :bookmark
         return true
-    elseif key == UInt32('r') || key == UInt32('R')
```

Comment:
One consideration of this PR would be that if we can remove <kbd>r</kbd> key from this Revise integration. @timholy can I hear your idea on this ?

Reply:
Yeah, let's not have Revise steal 2 keystrokes. I'm fine with giving `r` to `:remarks`